plugins/linux-pipewire/pipewire.c

```diff
@@ -732,8 +736,6 @@ static void on_process_cb(void *user_data)
 			obs_pw->cursor.width = bitmap->size.width;
 			obs_pw->cursor.height = bitmap->size.height;
 
-			g_clear_pointer(&obs_pw->cursor.texture,
-					gs_texture_destroy);
```

Comment:
Can you please add a `assert(obs_pw->cursor.texture == NULL)` here? :)

Reply:
Done!